.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing to CKAN
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the Comprehensive Kerbal Archive Network. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+* [Code of Conduct](#code-of-conduct)
+* [Reporting issues](#reporting-issues)
+  * [Reporting a bug](#reporting-a-bug)
+  * [Requesting a new feature](#requesting-a-new-feature)
+* [Helping with issues](#helping-with-issues)
+  * [Verifying issues](#verifying-issues)
+  * [Testing patches](#testing-patches)
+* [Translating](#translating)
+* [Creating pull requests](#creating-pull-requests)
+  * [Creating a patch](#creating-a-patch)
+  * [Creating a pull request](#Creating-a-pull-request)
+* [Coding conventions](#coding-conventions)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [CKAN Code of Conduct][1]. By participating, you are expected to uphold this code. Please report unacceptable behavior to mykdowling@gmail.com.
+
+## Reporting issues
+### Reporting a bug
+
+If you've found a problem in CKAN, do a search on GitHub under [Issues][2] in case it has already been reported.
+
+If you are unable to find any open GitHub issues addressing the problem you found, your next step will be to [open a new one][2] using the respective 'Bug' template.
+
+Your issue report should contain a title and as much relevant information as possible of the issue. Your goal should be to make it easy for yourself - and others - to reproduce the bug and figure out a fix.
+
+### Requesting a new feature
+
+If there's a new feature that you want to see added to CKAN, you can [open an issue][2] using the respective 'Feature' template, or you can write the code yourself.
+
+## Helping with issues
+### Verifying issues
+
+For starters, it helps just to verify bug reports. Can you reproduce the reported issue on your own computer? If so, you can add a comment to the issue saying that you're seeing the same thing.
+
+If an issue is very vague, can you help narrow it down to something more specific? Maybe you can provide additional information to help reproduce a bug, or help by eliminating needless steps that aren't required to demonstrate the problem.
+
+### Testing patches
+
+You can also help out by examining pull requests that have been submitted to CKAN via GitHub. In order to apply someone's changes, you need to first create a dedicated branch:
+
+`$ git checkout -b testing_branch`
+
+Then, you can use their remote branch to update your codebase. For example, let's say the GitHub user JohnSmith has forked and pushed to a topic branch "orange" located at https://github.com/JohnSmith/CKAN.
+
+```
+$ git remote add JohnSmith https://github.com/JohnSmith/CKAN.git
+$ git pull JohnSmith orange
```

Comment:
I just do a 'git checkout JohnSmith orange', without an extra branch and pull :)

Reply:
I always do `git fetch --all` and `git checkout -b mylocalname Who/what`.